.buckconfig

```diff
@@ -25,6 +25,10 @@
 [java]
     jar_spool_mode = direct_to_jar
     duplicates_log_level = fine
+    compile_against_abis = false
```

Comment:
Why?

Reply:
the plugin works in monorepo but okbuck tests are failing due to ASM in Buck trying to perform the class ABI on a java 17 class from a dependency. i'm working on a better fix